Boolean.md

```diff
@@ -0,0 +1,84 @@
+##Boolean Type
+Boolean is a fancy term meaning something that is either true or false.  While numbers and math form the backbone of any piece of software, everything (and I do mean everything) in computers, software and microchips, breaks down to ones and zeros, true and false.  While it isn't important to understand why that is and how exactly it works, it is important to understand how boolean expressions work so that your code will be able to do something interesting like this:
+
+````Swift
+func finishCourse() {
+    if gradeIsPassing() {
+    	print("Hooray!")
+   } else {
+      print("Doh!")
+   }
+}
+````
+
+The ````gradeIsPassing()```` function takes one course of action if a true value is returned and a different course of action if a false value is returned.
+Almost every program you write will need to take different courses of action based on boolean values, so it's important to have a really good grasp of what booleans and boolean expressions are as well as how to use them.
```

Comment:
If we go with the example I provided above. We can instead state something like...

----

Going with the example above, our program would like to know if `isHungry` is `true` or `false` and based upon that value, it will do something. If the person is hungry, it will feed them.. if not, it will do nothing.